SignalServiceKit/protobuf/Makefile

```diff
@@ -16,15 +16,23 @@ signal_service_protos: SignalService.proto
 provisioning_protos: Provisioning.proto
 	$(PROTOC) --swift_out=../src/Protos/Generated \
 		Provisioning.proto
+	$(WRAPPER_SCRIPT) --dst-dir=../src/Protos/Generated \
+		--wrapper-prefix=ProvisioningProto --proto-prefix=ProvisioningProtos --proto-file=Provisioning.proto
```

Comment:
This PR is a stepping stone to future work. It generates wrappers for the remaining schemas (but doesn't use them yet).